lib/shopify-cli/transform_data_structure.rb

```diff
@@ -0,0 +1,77 @@
+
+module ShopifyCli
+  ##
+  # `TransformDataStructure` helps to standardize data structure access. It
+  # traverses nested data structures and can convert
+  #
+  # * all strings used as keys to symbols,
+  # * all strings used as keys from `CamelCase` to `snake_case` and
+  # * associative array containers, e.g. from Hash to OpenStruct.
+  #
+  # Standardizing how a data structure is accessed can greatly reduces the risk
```

Comment:
nit: `can` seems to be a typo. Should be `...accessed greatly reduces` without the can